run-test

```diff
@@ -1,13 +1,35 @@
 #!/bin/bash -e
+# For coverage tests, run as
+#     coverage erase
+#     PATH=$PWD:$PATH RUNNER="coverage run --include=$PWD/**/*" ./run-test
+#     coverage html
+# See ./run-coverage
```

Comment:
Would something more simple like `RUNNER="coverage run --include=$PWD/**/*" ./run-test` without the lines before and afterwards work to demonstrate the RUNNER variable? The python3 example would also be nice to show.